pipelines/resources/tekton/task/func-s2i/0.1/func-s2i.yaml

```diff
@@ -55,7 +55,10 @@ spec:
       description: Digest of the image just built.
   steps:
     - name: generate
-      image: quay.io/openshift-pipeline/s2i:nightly
+      env:
+        - name: HOME
+          value: "/tekton/home/"
+      image: registry.redhat.io/ocp-tools-4-tech-preview/source-to-image-rhel8@sha256:98d8cb3a255641ca6a1bce854e5e2460c20de9fb9b28e3cc67eb459f122873dd
```

Comment:
One caveat: his image is just Technical Preview. Is it a problem? @lance @cardil @dsimansk

Reply:
I think it is not a problem, the same image is being used by OpenShift Pipelines team

Reply:
Is it the exact same image?

Reply:
Yes it is.